src/core/modules/SpecialMarkersMgr.cpp

```diff
@@ -131,38 +195,18 @@ void SpecialSkyMarker::draw(StelCore *core) const
 			break;
 		case FOV_RECTANGULAR:
 		{
-			QPoint a, b;
-			//const double fovRatio = qMax(angularSize[0], angularSize[1])/static_cast<double>(params.fov);
-			const double pixelsPerRad = static_cast<double>(prj->getPixelPerRadAtCenter());
-			const double width = pixelsPerRad * angularSize[0] * M_PI/180 ;
-			const double height = pixelsPerRad * angularSize[1] * M_PI/180 ;
-			QTransform transform = QTransform().translate(centerScreen[0], centerScreen[1]).rotate(-rotationAngle);
-			// bottom line
-			a = transform.map(QPoint(static_cast<int>(-width*0.5), static_cast<int>(-height*0.5)));
-			b = transform.map(QPoint(static_cast<int>(width*0.5), static_cast<int>(-height*0.5)));
-			sPainter.drawLine2d(a.x(), a.y(), b.x(), b.y());
-			// top line
-			a = transform.map(QPoint(static_cast<int>(-width*0.5), static_cast<int>(height*0.5)));
-			b = transform.map(QPoint(static_cast<int>(width*0.5), static_cast<int>(height*0.5)));
-			sPainter.drawLine2d(a.x(), a.y(), b.x(), b.y());
-			// left line
-			a = transform.map(QPoint(static_cast<int>(-width*0.5), static_cast<int>(-height*0.5)));
-			b = transform.map(QPoint(static_cast<int>(-width*0.5), static_cast<int>(height*0.5)));
-			sPainter.drawLine2d(a.x(), a.y(), b.x(), b.y());
-			// right line
-			a = transform.map(QPoint(static_cast<int>(width*0.5), static_cast<int>(height*0.5)));
-			b = transform.map(QPoint(static_cast<int>(width*0.5), static_cast<int>(-height*0.5)));
-			sPainter.drawLine2d(a.x(), a.y(), b.x(), b.y());
-
-			/*
-			 * NOTE: uncomment the code for display FOV value in top right corner of marker
-			if (fovRatio>=0.25)
```

Comment:
@alex-w shall this be permanently deleted?

Reply:
It's a "hidden feature" :) As far nobody asked for it before, so, this part may be removed.